tables/gurumuki.cti

```diff
@@ -20,191 +21,369 @@
 
 # This table is built and maintained under an activity of Braille Council of India
 # Contributors: Dipendra Manocha, Sreeja, Dinesh Kaushal, Mesar Hameed
-# Last updated on May 5, 2014
+# Updated on May 5, 2014
 # To report any bugs or any suggestion, please write to d@saksham.org and sreeja.param@gmail.com
 
+# Updated on May 12, 2022 by Jake Kyle, Compass Braille, jake@compassbraille.org
+
 include braille-patterns.cti
+include spaces.uti
```

Comment:
This rule made a test in spaces.yaml fail. This whole section should be moved to the new pa.yaml, and adapted if needed (or the table should be adapted to match the tests):

```yaml
table: pa.tbl
flags: {testmode: bothDirections}
tests:
  - ['\x0009', '9']
  - ['\x0020', '\x0020']
  - ['\x00a0', 'a']
flags: {testmode: forward}
tests:
  - ['\x000a', '\x0020']
  - ['\x000b', '\x0020']
  - ['\x000c', '\x0020']
  - ['\x000d', '\x0020']
  - ['\x2000', '\x0020']
  - ['\x2001', '\x0020']
  - ['\x2002', '\x0020']
  - ['\x2003', '\x0020']
  - ['\x2004', '\x0020']
  - ['\x2005', '\x0020']
  - ['\x2006', '\x0020']
  - ['\x2007', '\x0020']
  - ['\x2008', '\x0020']
  - ['\x2009', '\x0020']
  - ['\x200a', '\x0020']
  - ['\x2028', '\x0020']
  - ['\x2029', '\x0020']
  - ['\x202f', '\x0020']
  - ['\x205f', '\x0020']
```

Reply:
@bertfrees I think I can simply remove that rule from gurumuki.cti because it has the unwanted and unnecessary effect of overriding how spaces are already dealt with in en-chardefs.cti which is included in en-in-g1.ctb:
https://github.com/liblouis/liblouis/blob/83c91359ee77cabd49d64392c18cb2635855515c/tables/en-chardefs.cti#L25-L29